src/RenderWebGL.js

```diff
@@ -621,6 +621,60 @@ RenderWebGL.prototype._drawThese = function(
 };
 
 /**
+ * Draw say bubble next to the Drawable
+ * @param {int} drawableID A Drawable ID for draw bubble to
+ * @param {string} text A text for render
+ */
+
+RenderWebGL.prototype.renderBubble = function (drawableID, text) {
+    var  drawable = Drawable.getDrawableByID(drawableID)
+        ,bounds = drawable.getBounds()
```

Comment:
Before you call `getBounds`, you need to ensure that the convex hull of the Drawable has been updated. Maybe it would be preferable to call `this.getBounds`, which already does that.